modules/ROOT/pages/mule-4-api-javadoc.adoc

```diff
@@ -1,9 +1,20 @@
-= Mule API Javadoc
+= Mule Runtime Engine Javadoc
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: java, classes, javadoc, reference, objects, methods
 
-== Java Classes
+MuleSoft provides Javadocs for the Mule Kernel, the Community Edition of the Mule runtime engine.
+The Javadocs contain definitions for classes, fields and their data types, and also describes methods, including their usage, parameters and return values.
 
-* https://www.mulesoft.org/docs/site/4.1.1/apidocs/[Mule 4.1.1 API]
\ No newline at end of file
+Read the Mule SDK documentation if you want an easier way to extend the Mule Runtime Engine to meet your needs.
+
+You can find links to the Javadocs below:
+
+=== Java Classes
+
+* https://www.mulesoft.org/docs/site/4.1.1/apidocs/[Mule 4.1.1]
```

Comment:
Should this be updated to 4.2?

Reply:
Duke checked and 4.2 is not out there yet.